src/components/box/box.stories.tsx

```diff
@@ -285,3 +286,4 @@ export const Scroll: ComponentStory<typeof Box> = () => {
     </div>
   );
 };
+Scroll.parameters = { chromatic: { disable: true } };
```

Comment:
and for other as well

```suggestion
Scroll.parameters = { chromatic: { disableSnapshot: true } };
```